src/scripts/main.js

```diff
@@ -3,7 +3,43 @@
 const calendar = document.querySelector('#calendar');
 
 function calendarTable(year, month, element) {
-  // WRITE YOUR CODE HERE
+  const day = (new Date(year, month - 1).getDay() + 6) % 7;
```

Comment:
Главная проблема этой строки - сложность понимания

Reply:
What can I need to do? I only can write comments for explain. I don't see any possibilities about dividing this statement)). For me it's very clear. I need move order of day in calendar to one day,  because we have to implement solution with begin week from monday (position 0), whereas instance Date return position 0 for sunday.